### ISSUE TITLE
New release before Lexibank 2.0 releases?

### ISSUE THREAD

**Issue by chrzyki** [lexibank/pylexibank#277]
I think it would be worthwhile to do an update release of `pylexibank` before we push for (re-)releasing datasets for Lexibank 2.0. What do you think?

**Comment by LinguList** [comment]
Yes, I agree! Do we have the new consonant cluster checks in the code already?

**Comment by chrzyki** [comment]
> Yes, I agree! Do we have the new consonant cluster checks in the code already?

Yes. :) https://github.com/lexibank/pylexibank/pull/274
